nixops_gcp/nix/gce.nix

```diff
@@ -41,6 +41,46 @@ let
     '';
   };
 
+  imageOptions = { config, ...}: {
+    options = {
+      name = mkOption {
```

Comment:
would it be possible to factor these common options into a separate file `image-options.nix` and import that instead ?

Reply:
Well I thought about it and I considered adding it to the [gce-image.nix](https://github.com/nix-community/nixops-gce/blob/master/nixops_gcp/nix/gce-image.nix) but I've seen some differences and I didn't want to change the GCE image yet

Also, I see that there is also the _gceDiskOptions_ submodule that is defined in the same file

https://github.com/nix-community/nixops-gce/blob/3322656bd49cc30103547729856f102142669768/nixops_gcp/nix/gce.nix#L45-L48

Do you think we should move out both of them?

Reply:
I see the same name/family/project combination, what could be the difference between the 2 ?

Reply:
Done ; factored out : https://github.com/nix-community/nixops-gce/pull/9/commits/349d8b1ce1cc2229abe43270aa95f735bf3c1dc9